test/rest_v2.test.js

```diff
@@ -4,6 +4,56 @@ const { expect } = require("chai");
 const api = require("../lib/alpaca-trade-api");
 const mock = require("./support/mock-server");
 
+function checkTrade(trade) {
```

Comment:
It looks useful to move these to a helper function. A really minor thing, but I would call these functions like `assertTrade`, `assertBar`, etc.